Editor/Utility/EmbeddedAssetHelpers.cs

```diff
@@ -114,11 +102,14 @@ UnityEditor.VersionControl.Asset targetAsset
                             OnChanged(property.objectReferenceValue as T);
                     }
                     GUI.enabled = true;
-                    if (isLockedFile && GUILayout.Button("Check out"))
-                    {
-                        UnityEditor.VersionControl.Provider.Checkout(
-                            targetAsset, UnityEditor.VersionControl.CheckoutMode.Both);
-                    }
+                    if(m_Editor.target != null)
```

Comment:
Really strange indentation. Tabs vs spaces?

Reply:
Correct. Fixed the accident.